.github/workflows/release.yml

```diff
@@ -25,10 +25,19 @@ jobs:
         uses: cachix/install-nix-action@v16
         with:
           nix_path: nixpkgs=channel:nixos-21.11
-      - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v1
+      - uses: actions/cache@v2
```

Comment:
Nice! Should we also cache node_modules?

Reply:
Sounds good. Go with this PR as is and mark node npm followups